lib/auth/default_current_user_provider.rb

```diff
@@ -211,7 +211,7 @@ def current_user
     end
 
     # keep this rule here as a safeguard
-    # under no conditions to suspended or inactive accounts get current_user
+    # under no conditions to suspended, inactive or unapproved accounts get current_user
```

Comment:
```suggestion
    # under no conditions to suspended or inactive accounts get current_user
```
Let's revert this as well